src/manual/3rdparty/index.md

```diff
@@ -66,10 +66,21 @@ In particular, look for scopes of the forms:
  * `assume:mozilla-group:<group>`
  * `assume:mozillians-group:<group>`
 
-This is one of the rare cases where you should *not* apply scope expansion; that is, posessing `assume:mozillians-user:*` should not allow a user to see data for all mozillians!
+This is one of the rare cases where you should *not* apply scope expansion; that is, possessing `assume:mozillians-user:*` should not allow a user to see data for all Mozillians!
 
 See the [tc-login](/reference/core/login) reference documentation for more.
 
+# Don't be a [Confused Deputy](https://en.wikipedia.org/wiki/Confused_deputy_problem)
+
+If the service you are building acts on behalf of users, with its own TaskCluster credentials, you must be very careful to avoid allowing malicious users to abuse your privileges.
+For example, your service might create tasks based on a user's choices in a browser form.
+If the service has the scopes to create tasks that can read secrets, but does not verify that the user has such permission, then the service would provide a way for malicious users to create tasks that display those secrets.
+
+The best way to avoid this issue is to not act as a deputy.
+In this case, that would mean using the user's own TaskCluster credentials to create the tasks, rather than credentials assigned to the service.
+
+Where this is not possible, consult with the TaskCluster team before proceeding.
```

Comment:
Hmm... yes and no... `authorizedScopes` was specifically added to help people be _unconfused deputies_.

What you saying here is that services should be careful to avoid allowing scope elevation.